### DISCUSSION TITLE
Any way to check internet connection

### DISCUSSION THREAD

**Discussion by vinaygregoryjohn183** [alexbrazier/react-native-network-logger#85] [Q&A]
My use case is to check if the app is connected to the internet on a button press. Is this possible? Someone please help.

**Comment by alexbrazier** [comment]
This network logger is just used for displaying network requests that have been made by the app. You'd have to trigger the network request yourself and then verify it in the app. You would then be able to see the request in the network logger if you wanted.

e.g. something like this:

```tsx
const onPress = () => {
  fetch('/api/test').then((res) => {
    setConnected(res.ok);
  })
  .catch(() => {
    setConnected(false)
  })
}

**Reply by vinaygregoryjohn183** [reply]
@alexbrazier Got it. Thank you
